loadgen/prepare-loadgen.js

```diff
@@ -39,7 +40,10 @@ export async function prepareLoadgen(home, deployPowers) {
   let loadgenKit = await E(scratch).get(key);
   if (!loadgenKit) {
     const { bundleSource, pathResolve = pathResolveShim } = deployPowers;
-    const install = await makeInstall(home, deployPowers);
+    const mustPublish =
+      env.MUST_USE_PUBLISH_BUNDLE === '1' ||
+      env.MUST_USE_PUBLISH_BUNDLE === 'true';
```

Comment:
Unless you shoot for a mutual exclusion, the usual idiom is the inverse: empty string and undefined are false, but everything else is true.

Reply:
I dislike `ENV=false` and `ENV=0` being truthy. I normally have a helper for this, but it's not easily pullable from here (RESM code)